docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1,12 +1,378 @@
+usual. This action is suitable when certain values are expected to be
+consistently anomalous and they affect the model in a way that negatively
+impacts the rest of the results.
+
+`scope`::
+(object) An optional scope of series where the rule applies. A rule must either
+have a non-empty scope or at least one condition. By default, the scope includes
+all series. Scoping is allowed for any of the fields that are also specified in
+`by_field_name`, `over_field_name`, or `partition_field_name`. To add a scope
+for a field, add the field name as a key in the scope object and set its value
+to an object with the following properties:
+
+`filter_id`:::
+(string) The id of the filter to be used.
+
+`filter_type`:::
+(string) Either `include` (the rule applies for values in the filter) or
+`exclude` (the rule applies for values not in the filter). Defaults to `include`.
+
+`conditions`::
+(array) An optional array of numeric conditions when the rule applies. A rule
+must either have a non-empty scope or at least one condition. Multiple
+conditions are combined together with a logical `AND`. A condition has the
+following properties: 
+
+`applies_to`:::
+(string) Specifies the result property to which the condition applies. The
+available options are `actual`, `typical`, `diff_from_typical`, `time`.
+
+`operator`:::
+(string) Specifies the condition operator. The available options are `gt`
+(greater than), `gte` (greater than or equals), `lt` (less than) and `lte` (less
+than or equals).
+
+`value`:::
+(double) The value that is compared against the `applies_to` field using the
+`operator`.
+--
++
+--
+NOTE: If your detector uses `lat_long`, `metric`, `rare`, or `freq_rare`
+functions, you can only specify `conditions` that apply to `time`.
+
+--
+end::custom-rules[]
+
+tag::custom-settings[]
+Advanced configuration option. Contains custom meta data about the job. For
+example, it can contain custom URL information as shown in
+{stack-ov}/ml-configuring-url.html[Adding custom URLs to {ml} results].
+end::custom-settings[]
+
+tag::data-description[]
+The data description defines the format of the input data when you send data to
+the job by using the <<ml-post-data,post data>> API. Note that when configure
+a {dfeed}, these properties are automatically set.
++
+--
+When data is received via the <<ml-post-data,post data>> API, it is not stored
+in {es}. Only the results for anomaly detection are retained.
+
+A data description object has the following properties:
+
+`format`:::
+  (string) Only `JSON` format is supported at this time.
+
+`time_field`:::
+  (string) The name of the field that contains the timestamp.
+  The default value is `time`.
+
+`time_format`:::
+(string)
+include::{docdir}/ml/ml-shared.asciidoc[tag=time-format]
+--
+end::data-description[]
+
 tag::dependent_variable[]
-`dependent_variable`::
-(Required, string) Defines which field of the document is to be predicted. 
+Defines which field of the document is to be predicted. 
```

Comment:
I'm wondering if we can put it back until I start to work on the DFA related parameters? I'd like to be consistent, too, but then we need to delete the name and the info between parentheses in the case of dependent_variable, gamma, eta, lambda, feature_bag_fraction, maximum_number_trees, and so on, then open a PR that adds the removed info back to the API docs. Finally, merge and backport the two PRs (this one and the one that put the titles back to the API docs) in sync.
The other option is to leave it as-is for now and I'll remove the titles in this file and add them back in the API docs in one PR later. WDYT? If my train of thought is obscure and I explained my concern poorly, please let me know.

```suggestion
`dependent_variable`::
(Required, string) Defines which field of the document is to be predicted. 
```

Reply:
Agreed, there was no reason for me to change this, since it isn't used by any of the APIs I updated.